pkg/serverless/protocol.go

```diff
@@ -56,6 +76,98 @@ func StartDaemon(stopCh chan struct{}) *Daemon {
 	return daemon
 }
 
+// StartHTTPLogsServer starts an HTTP server, receiving logs from the AWS platform.
+// Returns the HTTP URL on which AWS should send the logs.
+// FIXME(remy): that would be awesome to have this directly running within the initial HTTP daemon?
```

Comment:
I guess that would make sense to put everything under `/lambda/...` and re-use the existing HTTP server.

Reply:
Finally took the time to address this 👍